specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-01-10/service.json

```diff
@@ -9025,8 +9110,7 @@
       "properties": {
         "instanceType": {
           "description": "Gets the class type.",
-          "type": "string",
-          "readOnly": true
```

Comment:
Why are we removing readOnly for instanceType? Is this something we wanted to do?